Gemfile

```
@@ -91,28 +85,29 @@ gem "anyway_config", "2.1.0"
 group :development, :test do
   gem 'aruba'
   gem 'ci_reporter_rspec'
-  gem 'conjur-cli', '~> 6.1'
-  gem 'conjur-debify', require: false
+  gem 'conjur-cli', git: 'https://github.com/cyberark/conjur-cli.git', branch: 'bump-version-6.2.6'
+  gem 'conjur-debify', github: 'conjurinc/debify', branch: 'ruby-3.0.2-upgrade'
   gem 'csr'
-  gem 'cucumber'
-  gem 'database_cleaner'
-  gem 'debase'
-  gem 'json_spec'
+  gem 'cucumber', '~> 7.1.0'
+  gem 'database_cleaner', '~> 1.8.4'
+  gem 'debase', '~> 0.2.5.beta2'
+  gem 'json_spec', '~> 1.1.5'
   gem 'net-ssh'
   gem 'parallel'
-  gem 'pry-byebug'
+  gem 'pry-byebug', '~> 3.9.0'
   gem 'pry-rails'
   gem 'rails-controller-testing'
   gem 'rails_layout'
   gem 'rake_shared_context'
   gem 'rspec'
-  gem 'rspec-core', '~> 3.0'
-  gem 'rspec-rails'
+  gem 'rspec-core'
+  gem 'rspec-rails', '~> 5.0.2'
   gem 'ruby-debug-ide'
   gem 'spring'
   gem 'spring-commands-cucumber'
   gem 'spring-commands-rspec'
   gem 'table_print'
+  gem 'rexml', '~> 3.2.5'
```

Is it really necessary? If so let's stick to major:
```suggestion
  gem 'rexml', '~> 3.2'
```